cmd/minikube/cmd/status.go

```diff
@@ -203,59 +204,63 @@ var statusCmd = &cobra.Command{
 		cname := ClusterFlagValue()
 		api, cc := mustload.Partial(cname)
 
-		var statuses []*Status
+		for {
+			var statuses []*Status
```

Comment:
Could you move all of the code in the for loop into a separate function? That would make this more readable.

Reply:
@priyawadhwa i moved the code to a separate func `writeStatusesAtInterval`, please have a look